docs/docker.rst

```diff
@@ -0,0 +1,79 @@
+.. _eprosima_docker_image:
+
+eProsima Docker Image
+=====================
+
+eProsima provides the eProsima XRCE-DDS Suite Docker image for those who want a quick demonstration of XRCE-DDS running on an Ubuntu
+platform. It can be downloaded from `eProsima's downloads page <https://eprosima.com/index.php/downloads-all>`_.
+
+This Docker image was built for Ubuntu 20.04 (Focal Fossa).
+
+To run this container you need Docker installed. From a terminal run
+
+.. code-block:: bash
+
+    $ sudo apt install docker.io
+
+.. _xrce_dds_suite:
+
+XRCE-DDS Suite
+--------------
+
+This Docker image contains the complete XRCE-DDS suite. This includes:
+
+- :ref:`eProsima XRCE-DDS libraries and examples <xrcedds_suite_examples>`: XRCE-DDS libraries bundled with several
+  examples that showcase a variety of capabilities of eProsima's Fast DDS implementation.
```

Comment:
In general, if you are not enforcing any code style regarding line length, then there is no need to break lines within a sentence.